packages/web-frontend/src/components/View/MatrixWrapper/components/HeaderRow.js

```diff
@@ -88,7 +89,11 @@ export default class HeaderRow extends PureComponent {
         {columns.slice(startColumn, displayedColumnCount).map(column => (
           <div key={column.key} style={styles.headerCell}>
             <div style={styles.clinicLabel}>
-              <div style={column.isGroupHeader ? styles.facilityTypeLabel : {}}>{column.title}</div>
+              {hideColumnTitles ? null : (
+                <div style={column.isGroupHeader ? styles.facilityTypeLabel : {}}>
+                  {column.title}
+                </div>
+              )}
```

Comment:
I added this config `hideColumnTitles` as we dont want to show the column titles for this matrix https://github.com/beyondessential/tupaia-backlog/issues/902#issue-656270589